astropy/modeling/utils.py

```diff
@@ -328,14 +328,14 @@ class AliasDict(MutableMapping):
     3
 
     Keys in the original parent dict are not visible if they were not
-    aliased::
```

Comment:
Why do these need only one? The following paragraph is a code block.

Reply:
My understanding is the doctest/python prompt ``>>>`` triggers a special case - just checked in a live sphinx editor. This is probably documented somewhere...